test/integration/helpers.js

```diff
@@ -171,18 +171,22 @@ if (process.env.HARDHAT_INTEGRATION_TESTS) {
     global.deployer = await ethers.getSigner('0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199') // last hardhat account
     global.superFluidDeployer = await ethers.getSigner('0xdD2FD4581271e230360230F9337D5c0430Bf44C0') // penultimate hardhat account
 
-    fetchNeededTokens()
+    await fetchNeededTokens()
```

Comment:
```suggestion
    fetchNeededTokens()
```
This one can be without an `await` because of the slow superfluid initialize all the contracts are loaded before SF ends =) or you hit an issue with this one?

Reply:
Oh no, this was "just in case it was missed" since I saw it was an async method.

Was one of the tests when trying to figure it out why the Superfluid framework didn't start =)